pkg/deployments/tasks/goerli-tokens-factories/input.js

```diff
@@ -0,0 +1,235 @@
+  stake: {
+    contractName: "AuthdMockToken",
+    contract: "@sense-finance/v1-core/src/tests/test-helpers/mocks/MockToken.sol:AuthdMockToken",
+    name: "WETH",
+    symbol: "WETH",
+    decimals: 18,
+    address: WETH_TOKEN.get(CHAINS.GOERLI),
+  },
+  reward: {
+    contractName: "AuthdMockToken",
+    contract: "@sense-finance/v1-core/src/tests/test-helpers/mocks/MockToken.sol:AuthdMockToken",
+    name: "Sense Reward Token",
+    symbol: "SRT",
+    decimals: 18,
+    address: REWARD_TOKEN.get(CHAINS.GOERLI),
+  },
+};
+
+const GOERLI_FACTORIES = [
+  {
+    address: "0xDbE7deE84f9A32E5b2CA8215ea6Ad85b4c293655",
+    contract: "MockCropFactory",
+    contractName: "MockCropFactory",
+    ifee: ethers.utils.parseEther("0.0010"), // 0.1%
+    stake: GOERLI_TOKENS.stake.address,
+    stakeSize: ethers.utils.parseEther("0.05"), // 0.05 WETH
+    minm: 0, // 0 days
+    maxm: (10 * 365.25 * 24 * 60 * 60).toString(), // 10 years
+    mode: 1, // 1 = weekly
+    tilt: 0,
+    guard: ethers.utils.parseEther("100000"), // $100'000 (not used in Goerli)
+    targets: [GOERLI_TOKENS.tokens[0].target, GOERLI_TOKENS.tokens[1].target, GOERLI_TOKENS.tokens[2].target],
+    maturities: MATURITIES,
+    reward: GOERLI_TOKENS.reward,
+  },
+  {
+    address: "0x27F6f7E23b4D77FE9aD7f9A0756a9b40bddE32c9",
+    contract: "Mock4626CropFactory",
+    contractName: "Mock4626CropFactory",
+    ifee: ethers.utils.parseEther("0.0010"), // 0.1%
+    stake: GOERLI_TOKENS.stake.address,
+    stakeSize: ethers.utils.parseEther("0.05"), // 0.05 WETH
+    minm: 0, // 0 days
+    maxm: (10 * 365.25 * 24 * 60 * 60).toString(), // 10 years
+    mode: 1, // 1 = weekly
+    tilt: 0,
+    guard: ethers.utils.parseEther("100000"), // $100'000 (not used in Goerli)
+    targets: [GOERLI_TOKENS.tokens[1].target4626, GOERLI_TOKENS.tokens[2].target4626],
+    maturities: MATURITIES,
+    reward: GOERLI_TOKENS.reward,
+  },
+];
+
+const GOERLI_ADAPTERS = [
+  {
+    isOwnable: true,
+    address: "0x7b6da076144b0a9cf0cddf908f605924d9e0a180",
+    contractName: "MockOwnableAdapter",
+    ifee: ethers.utils.parseEther("0"), // 0%, no issuance fees
+    oracle: ethers.constants.AddressZero,
+    stake: WETH_TOKEN.get(CHAINS.GOERLI),
+    stakeSize: ethers.utils.parseEther("0.25"), // 0.25 WETH
+    minm: 0, // 0 days
+    maxm: (10 * 365.25 * 24 * 60 * 60).toString(), // 10 years
+    tilt: 0,
+    level: 31,
+    mode: 0, // 1 = weekly
+    target: GOERLI_TOKENS.tokens[0].target,
+    maturities: [],
+  },
+  {
+    isOwnable: true,
+    address: "0xA1Dc1E9C1a2a087874634cc7D0395658A741027c",
+    contractName: "MockOwnableAdapter",
+    ifee: ethers.utils.parseEther("0"), // 0%, no issuance fees
+    oracle: ethers.constants.AddressZero,
+    stake: WETH_TOKEN.get(CHAINS.GOERLI),
+    stakeSize: ethers.utils.parseEther("0.25"), // 0.25 WETH
+    minm: 0, // 0 days
+    maxm: (10 * 365.25 * 24 * 60 * 60).toString(), // 10 years
+    tilt: 0,
+    level: 31,
+    mode: 0, // 1 = weekly
```

Comment:
```suggestion
    mode: 1, // 1 = weekly
```
(same with the other adapters)

Reply:
yep, it's actually mode: 0, // 0 = monthly since this is an ownable adapter and must be 0